deps/build.log

```diff
@@ -0,0 +1,16 @@
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100    85  100    85    0     0    464      0 --:--:-- --:--:-- --:--:--   461
+
100  794k  100  794k    0     0  3933k      0 --:--:-- --:--:-- --:--:-- 3933k
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100    71  100    71    0     0    353      0 --:--:-- --:--:-- --:--:--   353
+
100 55483  100 55483    0     0   255k      0 --:--:-- --:--:-- --:--:--  255k
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100    79  100    79    0     0    387      0 --:--:-- --:--:-- --:--:--   389
+
100 88197  100 88197    0     0   377k      0 --:--:-- --:--:-- --:--:--  377k
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100    84  100    84    0     0    497      0 --:--:-- --:--:-- --:--:--   497
+
100 91448  100 91448    0     0   493k      0 --:--:-- --:--:-- --:--:--  493k
```

Comment:
Minor nit pick but maybe this should be omitted and in .gitignore?

Reply:
Done.